### DISCUSSION TITLE
https://notes.junghanacs.com/notes/20240404T115445

### DISCUSSION THREAD

**Discussion by giscus** [junghanacs/notes.junghanacs.com#13] [Announcements]
# https://notes.junghanacs.com/notes/20240404T115445

What is the fediverse? 페디버스는 분산형 소셜네트워크로, 서로 연결된 다양한 플랫폼에서 사용자가 자유롭게 소통할 수 있는 공간입니다. #정한: #김정한 - 기예인 텍마 fosstodon.org/@chanjin65@threads.net FOSSTODON 여기가 메인이다.

https://notes.junghanacs.com/notes/20240404T115445

**Comment by junghan0611** [comment]
트위터 혹한다 블루스카이 이든,뭐든 혹한다 좋지 않다
